apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -990,6 +990,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     * @param _setting Setting instance applicable to the vote
     * @return True if the vote's execution delay has not finished
     */
+   // Note: seems a bit weird to say "not", even if it is a minor optimization
```

Comment:
@facuspagnuolo probably a quick one—not sure how you feel about this.

Reply:
Yup, renamed in 09317601554141b9d29c187674380e8dd22389a9